.taskcluster.yml

```diff
@@ -45,7 +45,7 @@ tasks:
                cd bugbug &&
                git checkout ${head_rev} &&
                pip install -r test-requirements.txt &&
-               flake8"
+               pre-commit run -a" # TODO Be smarter and run only on modified files
```

Comment:
Please file an issue about this, so we don't forget.

Reply:
Done https://github.com/mozilla/bugbug/issues/298